packages/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -96,6 +103,62 @@ def is_fim_scan_ended():
     return -1
 
 
+def create_file(type, path):
+    """ Creates a file in a given path.
+
+    :param type: Defined constant that specifies the type. It can be: FIFO, SYSLINK, SOCKET or REGULAR
+    :param path: Path where the file will be created
+    """
+    getattr(sys.modules[__name__], f'_create_{type}')(path)
+
+
+def _create_fifo(path):
+    fifo_path = os.path.join(path, 'fifo_file')
+    try:
+        os.mkfifo(fifo_path)
+    except OSError:
+        raise
+
+
+def _create_sys_link(path):
+    syslink_path = os.path.join(path, 'syslink_file')
+    try:
+        os.symlink(syslink_path, syslink_path)
+    except OSError:
+        raise
+
+
+def _create_socket(path):
+    socket_path = os.path.join(path, 'socket_file')
+    try:
+        os.unlink(socket_path)
+    except OSError:
+        if os.path.exists(socket_path):
+            raise
+    sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+    sock.bind(socket_path)
+
+
+def _create_regular(path):
+    regular_path = os.path.join(path, 'regular_file')
+    with open(regular_path, 'w') as f:
+        f.write('')
+
+
+def change_internal_options(int_opt_path, pattern, value):
```

Comment:
This function should allow creating new params too. As a user, I would like to pass a param name and a value and the function should manage it.

Reply:
Done